RELEASE.md

```diff
@@ -1,26 +1,69 @@
+# *CDM Model - Equity Products*
 
 _Background_
 
-The Asset Refactoring initiative (see [#2805](https://github.com/finos/common-domain-model/issue/2805)) is seeking to improve the Product Model to address some long-standing issues and to ensure the continued extensibility to additional financial products and markets. A proposal has been agreed - through a cross-industry Task Force - to implement this remodelling in the CDM.
-
-This release includes two minor adjustments following three planned major tranches of work in CDM 6 to implement the refactored model.
+This release contains modifications required to accomodate Equity and Exotic Products under individual Asset Classes
 
 _What is being released?_
 
-An optional attribute exists on the `AssetPayout` data type to uniquely identify the different types of securities financing trade types, such as a repurchase transaction or a buy/sell-back.  The name of this attribute has been updated to have a broader potential scope and not limit just to repos.  Specifically, the attribute `repoType` has been renamed to `tradeType` and its data type has been renamed from `RepoTypeEnum` to `AssetPayoutTradeTypeEnum`.  The values of the enumerator have not been changed.
+This release creates following modifications:
+- a new qualification function `Qualify_Equity_OtherOption` to add qualification for Exotic Options
+```
+func Qualify_Equity_OtherOption: <"Qualifies a product with properties of an Exotic Option as Equity Option (Other)">
+    [qualification Product]
+    inputs:
+        economicTerms EconomicTerms (1..1)
+    output:
+        is_product boolean (1..1)
+            [synonym ISDA_Taxonomy_v1 value "Qualify_EquityOther"]
+            [synonym ISDA_Taxonomy_v2 value "Qualify_EquityOther"]
+    set is_product:
+        Qualify_AssetClass_Equity(economicTerms) = True
+            and economicTerms -> payout -> optionPayout only exists
+            and economicTerms -> nonStandardisedTerms = True
+```
+- changes in existing options to avoid duplicate qualification of options
+```
+e.g.
+func Qualify_EquityOption_PriceReturnBasicPerformance_SingleName
+now contains additional clause
+and (if economicTerms -> nonStandardisedTerms exists then economicTerms -> nonStandardisedTerms = False else True)
+```
+
+- modification in FpML conditions `FpML_ird_9` and `FpML_ird_29`
+```
+    condition FpML_ird_9: <"FpML validation rule ird-9 - If calculationPeriodAmount/calculation/compoundingMethod exists, then resetDates must exist.">
+        if compoundingMethod exists and compoundingMethod <> CompoundingMethodEnum -> None then resetDates exists
+    condition FpML_ird_29: <"FpML validation rule ird-29 - If compoundingMethod exists, then fixedRateSchedule must not exist.">
+        if compoundingMethod exists and compoundingMethod <> CompoundingMethodEnum -> None
+        then rateSpecification -> FixedRateSpecification is absent
+```
+- relaxation of cardinality rule for `expirationTime`
```

Comment:
Do not include snippets. Instead, explain how the cardinalities of 2 attributes have changed (one relaxed, one tightened), and how the new condition makes everything hangs together.

The change should be provided in a dedicated backward-incompatible section, and should point to at least 1 representative sample (which can be looked at in the PR) that demonstrates how an implementor should update any previously persisted object so that it is valid again.